webpack.config.js

```diff
@@ -28,7 +28,7 @@ module.exports = [
   {
     entry: './demo/demo.jsx',
     output: {
-      path: `${__dirname}/demo`,
+      path: `${__dirname}/`,
```

Comment:
The index.html imports the file from `./demo.js` not `./demo/demo.js`
![image](https://user-images.githubusercontent.com/29656850/115025519-d5412a00-9e97-11eb-8b79-48fc71cc3d80.png)

Reply:
Please, move this change to another PR, because it is not related to the feature proposed here